src/ui/Views/Course/Variants.cshtml

```diff
@@ -2,9 +2,10 @@
 
 @{
     Layout = "_Layout";
+    var link = "mailto:becomingateacher@digital.education.gov.uk?subject=Problem%20with%20ucas%20data%20" + Model.Course.ProgrammeCode;
```

Comment:
Should be `"Problem with course" +`

Reply:
Sorry @defong, what @fofr  said, I gave you slightly wrong steer.